<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZerocoinPage</class>
 <widget class="QWidget" name="ZerocoinPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>455</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Right-click to edit address or label</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="denomination">
       <property name="text">
        <string>Select Denomination</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="zerocoinAmount"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="toolTip">
        <string>Export the data in the current tab to a file</string>
       </property>
       <property name="text">
        <string>&amp;Export</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../res/icons/export.png</normaloff>../res/icons/export.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>138</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Mint</string>
        </attribute>
        <widget class="QPushButton" name="zerocoinMintButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>123</width>
           <height>25</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Mint Zerocoin</string>
         </property>
         <property name="text">
          <string>&amp;Mint Zerocoin</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../res/icons/add.png</normaloff>../res/icons/add.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Spend</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>812</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Spend To</string>
         </property>
        </widget>
        <widget class="QPushButton" name="zerocoinSpendButton">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>134</width>
           <height>25</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>Spend Zerocoin</string>
         </property>
         <property name="text">
          <string>&amp;Spend Zerocoin</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../res/icons/edit.png</normaloff>../res/icons/edit.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="spendToThirdPartyAddress">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>10</y>
           <width>701</width>
           <height>25</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
        <widget class="QCheckBox" name="zerocoinSpendToMeCheckBox">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>810</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Spend To Me</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
